stages/org.osbuild.systemd.preset

```diff
@@ -28,14 +28,31 @@ SCHEMA = r"""
       }
     },
     "description": "Array of systemd unit names and their preset logic."
+  },
+  "user-presets": {
```

Comment:
just as a suggestions on types, @achilleas-k et all feel free to disagree, I would do

```
presets:
  system:
    - []
  user:
    - []
```

Reply:
I suggested a similar thing, but it is not as easy as you outlined. We would have to keep the old schema working for backward compatibility reasons. It must be possible to build old manifests which use this stage also with newer osbuild. We would have to keep the old schema and have it `oneOf` with the new schema. The stage would then had to handle both cases.

We ended up with this`user-presets` approach as the least disruptive and the least amount of work.